src/Fixer/Alias/BacktickToShellExecFixer.php

```diff
@@ -53,7 +53,12 @@ public function getDefinition(): FixerDefinitionInterface
     /**
      * {@inheritdoc}
      *
-     * Must run before EscapeImplicitBackslashesFixer, ExplicitStringVariableFixer, NativeFunctionInvocationFixer, SingleQuoteFixer.
```

Comment:
moving back to the original goal of this PR.

@localheinz , for me those comments are having almost no value anymore. They are not enough, we set up different approaches to have the priorities tested, and the comment here with 50 fixers listed is not useful (for me) anyway.

With that, if those comments are not useful for you, I suggest to drop this PR in the end, and we should have better solution (like discussed in other thread) for long run.
Yet, if those comments are bringing any value to you, come up with approach that is the best for you and I will gladly merge it.

Reply:
Then let's leave everything as it is or remove the comments entirely, @keradus - what do you think?

I only stumbled across this issue with the list of fixers in a separate pull request, took a note, and figured it would be easier to maintain if it was a vertical list (can easily move a line of code up or down in PhpStorm) rather than a horizontal list.

For me, these comments provide no value at the moment.

Reply:
If the list provides no value, there is no value to change format of that list. But it is also work already done here to do so, thus no extra effort.
Then, dropping the list at all would actually bring some new effort, maybe not needed for now.

Please, decide and apply the conclusion.